src/struct/AkairoHandler.js

```diff
@@ -113,6 +113,11 @@ class AkairoHandler extends EventEmitter {
             mod.category = dirs[dirs.length - 1];
         }
 
+        if (mod.aliases && mod.aliases.length === 0 && this.client.akairoOptions.automateAliases) {
```

Comment:
AkairoHandler is for all module types. You want to move this to CommandHandler.

Reply:
Ah, okay, I'll fix that